Makefile

```diff
@@ -30,8 +30,13 @@ clean-deps:
 start-deps:
 	docker compose up -d integration-deps
 
+start-deps-local:
```

Comment:
Don't understand why we need this and what it has to do with docs.

Reply:
Running stablesats with a local galoy backend. We don't need to have redis and otel running since we already have them provisioned with galoy.